internal/chezmoi/gpg.go

```diff
@@ -28,8 +30,12 @@ func (g *GPG) Decrypt(filename string, ciphertext []byte) ([]byte, error) {
 		return nil, err
 	}
 
+	if g.Command == "" {
+		g.Command = "gpg"
+	}
```

Comment:
Please set the default command to be `gpg` in [this function](https://github.com/twpayne/chezmoi/blob/master/cmd/config.go#L129) rather than using `if` statements here.

Reply:
I moved it, but I am still getting `Subprocess launched with function call as argument or cmd arguments (gosec)` error from the CI. I am still very new to go, could you please help me to fix it?

Thank you for review!

Reply:
Of course!

This warning comes from a very strict linter that gets run as part of chezmoi's build process. This linter spots things which are valid Go but might cause unintended problems. In this case, it's warning that we're launching a command (subprocess) where the command to launch is controlled by a variable. This means that if someone can change the value of the variable they can change which command is launch. This might be a security problem in a web service but of course here it's exactly what what we want, so we should disable the lint warning for this line.

The way to do this is to put on the line before:

```go
//nolint:gosec
cmd := exec.Command(
	gpg.Command,
```

This disables the gosec linter for this single line.

chezmoi uses a lot of strict automation that makes it more solid but also makes it a bit intimidating to contribute to. Thanks for persisting with this :)

Reply:
Thank you!

Hopefully everything is ok now :).